src/encoded/types/file.py

```diff
@@ -378,6 +382,59 @@ def create(cls, registry, uuid, properties, sheets=None):
             sheets['external'] = external_creds(bucket, key, name, profile_name)
         return super(File, cls).create(registry, uuid, properties, sheets)
 
+    def _get_external_sheet(self):
+        external = self.propsheets.get('external', {})
+        if external.get('service') == 's3':
+            return external
+        else:
+            raise HTTPNotFound()
+
+    def set_public_s3(self):
+        external = self._get_external_sheet()
+        boto3.resource('s3').ObjectAcl(
+            external['bucket'],
+            external['key']
+        ).put(ACL='public-read')
```

Comment:
where are 'public-read' and 'private' defined?  Are these base-level named ACLs in S3?

Reply:
Yes, these are ACLs that Amazon has defined.